package.json

```diff
@@ -99,7 +99,7 @@
     "file-loader": "^4.2.0",
     "gh-pages": "^2.1.1",
     "husky": "^3.0.9",
-    "jest": "23.6.0",
+    "jest": "^25.2.0",
```

Comment:
I thought upgrading jest would help with the timing tests but it didn't. But also I upgraded jest and the failure consoles are soooo much better. So I kept it.